### PR TITLE
Firelemon explosion is now delayed

### PR DESCRIPTION
**Pull request by XDTM**
This was most likely the intended behaviour since it already set throwing mode on, despite instantly exploding. The delay is between 1 and 6 seconds to make them very unreliable.

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
ma44: *heavy breathing*

**Comment by ma44** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: code/modules/hydroponics/grown/citrus.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
 		C.throw_mode_on()
-	prime()
```
Comment (Supermichael777) [inline code/modules/hydroponics/grown/citrus.dm]: put the delay in its prime

**Comment by XDTM** [comment]
I wanted to stay consistent with grenade code where prime() is the explosion itself

**Pull request merged**